File: Student.py
class Student:

    # class varaibles
    class_year = 2025
    num_student = 0

    def __init__(self, name, age):
        self.name = name
        self.age = age
        Student.num_student += 1

student1 = Student("Spongebob", 23)
student2 = Student("Patrick", 20)
student3 = Student("Squid", 55)
student4 = Student("Sandy", 27)


print(f"My graduating class of {Student.class_year} has {Student.num_student} students")
print(student1.name)
print(student2.name)
print(student3.name)
print(student4.name)

File: Student.java
public class Student{
    String name;
    int age;
    double gpa;
    boolean isStudent;

    Student(String name,int age,double gpa){
        this.name = name;
        this.age = age;
        this.gpa = gpa;
        this.isStudent = true;
    }

    void study(){
        System.out.println(this.name+ " is studing.");
    }
        

}